sample-apps/deepedit/lib/train.py

```diff
@@ -129,7 +128,8 @@ def val_pre_transforms(self, context: Context):
             LoadImaged(keys=("image", "label"), reader="nibabelreader"),
             SingleLabelSingleModalityd(keys=("image", "label")),
             AddChanneld(keys=("image", "label")),
-            Spacingd(keys=["image", "label"], pixdim=self.target_spacing, mode=("bilinear", "nearest")),
+            # Spacing might not be needed as resize transform is used later.
+            # Spacingd(keys=["image", "label"], pixdim=self.target_spacing, mode=("bilinear", "nearest")),
```

Comment:
why is this added and commented instead of just being removed?

Reply:
As a reference... just in case a researcher wants to use this transforms